src/server/_query.py

```diff
@@ -235,7 +235,7 @@ def run_query(p: APrinter, query_tuple: Tuple[str, Dict[str, Any]]):
     query, params = query_tuple
     # limit rows + 1 for detecting whether we would have more
     full_query = text(f"{query} LIMIT {p.remaining_rows + 1}")
-    current_user.log_info("full_query: %s, params: %s", full_query, params)
+    current_user.log_info({"full_query" : str(full_query), "params" : params})
```

Comment:
Use kwargs instead of logging a dictionary for all structlog logging

```suggestion
    current_user.log_info("Running query", full_query=str(full_query), params=params)
```

(& please propagate this change to other instances below)

Reply:
Thanks, changed the parameters into kwargs format and propagated to below